tests/support/util.tcl

```diff
@@ -682,20 +682,25 @@ proc string2printable s {
     return $res
 }
 
-# Check that probability of each element are between {min_prop} and {max_prop}.
-proc check_histogram_distribution {res min_prop max_prop} {
+# Calculation value of Chi-Square Distribution. By this value
+# we can verify the random distribution sample confidence.
+# Based on the following wiki:
+# https://en.wikipedia.org/wiki/Chi-square_distribution
+proc chi_square_value {res} {
```

Comment:
i would like to better document the inputs and output (scale / units or alike) of that function, so the reader doesn't need to read it or follow the link.